### PR TITLE
chore: get mocha tests to pass.

### PR DESCRIPTION
**Pull request by BeksOmega**
## The basics

- [ ] I branched from develop
- [ ] My pull request is against develop
- [ ] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)
- [ ] I ran `npm run format` and `npm run lint`

### Resolves

N/A

### Proposed Changes

Does various things to get the mocha tests to pass. See each individual commit.

#### Behavior Before Change

Mocha tests do not pass.

#### Behavior After Change

Mocha tests do pass.

### Reason for Changes

CI is important

### Test Coverage

This is tests silly!

### Documentation

N/A

### Additional Information

We are getting some console errors (which are not causing tests to fail) that I need to investigate. But I wanted to put this up to see if CI passes.

### CODE REVIEW

File: core/gesture.ts [hunk 1]
```diff
@@ -169,10 +169,10 @@ export class Gesture {
 
   /**
    * @param e The event that kicked off this gesture.
-   * @param creatorWorkspace The workspace that created this gesture and has a
+   * @param creatorWorkspace_ The workspace that created this gesture and has a
    *     reference to it.
    */
-  constructor(e: Event, private readonly creatorWorkspace: WorkspaceSvg) {
+  constructor(e: Event, private readonly creatorWorkspace_: WorkspaceSvg) {
```
Comment (gonfunko) [inline core/gesture.ts:175]: Was the old name conflicting or something? I think we're generally not wanting to use underscores to indicate private-ness.

**Review by gonfunko — APPROVED** [review]

File: core/gesture.ts (hunk 1 above)
Comment (BeksOmega) [reply core/gesture.ts]: We had a test checking for `creatorWorkspace_` idk why I decided to fix it this way. Reverted this and modified the test instead.

**Comment by BeksOmega** [comment]
@gonfunko can you give this another look? Mainly just want opinions on making the workspace nullable.

**Review by gonfunko — APPROVED** [review]

**Comment by BeksOmega** [comment]
We discussed that the workspace nullability is not great, but we want to unblock investigating source maps, and doing development in `develop`. So we're merging this for now, and Aaron is going to investigate removing the nullability.

**Pull request merged**